pkg/bluge/search/search.go

```diff
@@ -72,17 +92,31 @@ func MultiSearch(ctx context.Context, query *meta.ZincQuery, mappings *meta.Mapp
 		if err != nil {
 			return nil, err
 		}
+		if docList.sort == nil {
+			if req, ok := req.(*bluge.TopNSearch); ok {
+				docList.sort = req.SortOrder().Copy()
```

Comment:
Don't we sort for `bluge.AllMatches` as well?

Reply:
In zinc, we just use topN search.

Reply:
okay